pipelines/configs/seqr.yml

```diff
@@ -1,12 +1,17 @@
 namespace: main
 analysis_dataset: seqr
-skip_datasets: ['ohmr3-mendelian', 'ohmr4-epilepsy', 'mito-disease', 'flinders-ophthal']
+#skip_datasets: ['ohmr3-mendelian', 'ohmr4-epilepsy', 'mito-disease', 'flinders-ophthal']
 skip_samples: [CPG11783,CPG12310,CPG199679,CPG13326]  # acute-care,acute-care,acute-care,perth-neuro
+datasets: ['ohmr3-mendelian', 'ohmr4-epilepsy']  # ['heartkids', 'ravenscroft-rdstudy']  # ['circa', 'acute-care', 'perth-neuro', 'heartkids', 'ravenscroft-rdstudy']
+check_inputs: true
 keep_scratch: true
 skip_samples_with_missing_input: false
-ped_checks: false
-version: may21
+version: 'may21'
```

Comment:
What does this version mean? It's may22 btw, could it be automatically generated?

Reply:
I meant "21 May", as in day, not year, haha.
New version means that the workflow wouldn't attempt to reuse intermediate results from previous runs (except for CRAMs and GVCFs). With the same version, intermediate joint-calling, VQSR, matrix tables, etc would be reused if exist. I should probably move into the direction of checking if the inputs changed (i.e. the set of input samples and input FASTQ changed) instead of these strict versions, but that's for another PR.